tests/robot-tests/tests/admin_and_public/bau/publish_content.robot

```diff
@@ -0,0 +1,93 @@
+*** Settings ***
+Library             ../../libs/admin_api.py
+Resource            ../../libs/admin-common.robot
+Resource            ../../libs/admin/manage-content-common.robot
+Resource            ../../libs/public-common.robot
+
+Suite Setup         user signs in as bau1
+Suite Teardown      user closes the browser
+
+Force Tags          Admin    Local    Dev    AltersData
+
+*** Variables ***
+${PUBLICATION_NAME}=    UI tests - publish content %{RUN_IDENTIFIER}
+${RELEASE_NAME}=        Calendar Year 2001
+
+*** Test Cases ***
+Create new publication and release via API
+    ${PUBLICATION_ID}=    user creates test publication via api    ${PUBLICATION_NAME}
+    user create test release via api    ${PUBLICATION_ID}    CY    2001
+
+Navigate to release content
+    user navigates to editable release summary from admin dashboard    ${PUBLICATION_NAME}
+    ...    ${RELEASE_NAME} (not Live)
+    user clicks link    Content
+    user waits until page contains button    Add new section
+
+Add an accordion section to release content
+    user clicks button    Add new section
+    user changes accordion section title    1    Test section
+
+Add text block with link to absence glossary entry to accordion section
+    user adds text block to editable accordion section    Test section    css:#releaseMainContent
+    #user adds content to accordion section text block    Test section    1    Test. <a href=%{PUBLIC_URL}/glossary#absence>Absence</a>. Test.    css:#releaseMainContent
```

Comment:
Might as well remove the commented-out text unless you want to revisit later?  I'm guessing it's superceded by the following steps though now.